frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/confirmation.tsx

```diff
@@ -131,25 +130,104 @@ export async function loader({ context: { session }, params, request }: LoaderFu
     acessToDentalInsurance: state.adultState.dentalInsurance,
     selectedFederalBenefits,
     selectedProvincialBenefits,
+  };*/
+
+  const userInfo = {
+    firstName: 'firstName',
+    lastName: 'lastName',
+    phoneNumber: '1112223333',
+    altPhoneNumber: '1112223333',
+    preferredLanguage: '1033',
+    birthday: toLocaleDateString(parse('1990-11-11', 'yyyy-MM-dd', new Date()), locale),
+    sin: '800000002',
+    martialStatus: 'MARRIED',
+    email: 'EMAIL',
+    communicationPreference: {
+      confirmEmail: 'confirmEmail',
+      email: 'email',
+      preferredLanguage: '1033',
+      preferredMethod: 'EMAIL',
+    },
+  };
+  const spouseInfo = state.adultChildState.partnerInformation
+    ? {
+        firstName: 'firstName',
+        lastName: 'lastName',
+        birthday: toLocaleDateString(parse('1990-11-11', 'yyyy-MM-dd', new Date()), locale),
+        sin: '700000003',
+        consent: true,
+      }
+    : undefined;
+
+  const mailingAddressInfo = {
+    address: '111 Main St',
+    city: 'Ottawa',
+    province: { provinceTerritoryStateId: 'daf4d05b-37b3-eb11-8236-0022486d8d5f', countryId: '0cf5389e-97ae-eb11-8236-000d3af4bfc3', nameEn: 'Ontario', nameFr: 'Ontario', abbr: 'ON' },
+    postalCode: 'K1K2H2',
+    country: { countryId: '0cf5389e-97ae-eb11-8236-000d3af4bfc3', nameEn: 'Canada', nameFr: 'Canada' },
+    apartment: '',
+  };
+
+  const homeAddressInfo = {
+    address: '111 Main St',
+    city: 'Ottawa',
+    province: { provinceTerritoryStateId: 'daf4d05b-37b3-eb11-8236-0022486d8d5f', countryId: '0cf5389e-97ae-eb11-8236-000d3af4bfc3', nameEn: 'Ontario', nameFr: 'Ontario', abbr: 'ON' },
+    postalCode: 'K1K2H2',
+    country: { countryId: '0cf5389e-97ae-eb11-8236-000d3af4bfc3', nameEn: 'Canada', nameFr: 'Canada' },
+    apartment: '',
+  };
+
+  const dentalInsurance = {
+    acessToDentalInsurance: true,
+    selectedFederalBenefits: {
+      access: true,
+      benefit: '758bb862-26c5-ee11-9079-000d3a09d640',
+    },
+    selectedProvincialBenefits: {
+      access: true,
+      province: '9c440baa-35b3-eb11-8236-0022486d8d5f',
+      benefit: 'b3f25fea-a7a9-ee11-a569-000d3af4f898',
+    },
+  };
+
+  const childInfo = {
+    firstName: 'firstName',
+    lastName: 'lastName',
+    birthday: toLocaleDateString(parse('2009-11-11', 'yyyy-MM-dd', new Date()), locale),
+    sin: '800000002',
+    isParent: true,
+    dentalInsurance: {
+      acessToDentalInsurance: true,
+      federalBenefit: {
+        access: true,
+        benefit: '758bb862-26c5-ee11-9079-000d3a09d640',
+      },
+      provTerrBenefit: {
+        access: true,
+        province: '9c440baa-35b3-eb11-8236-0022486d8d5f',
+        benefit: 'b3f25fea-a7a9-ee11-a569-000d3af4f898',
+      },
+    },
```

Comment:
What is all of this hardcoded data?

Reply:
Talked to @AdamP-HRSDC-RHDCC in a side chat. It's just dummy data that will eventually be removed. Resolving.

Reply:
(from Slack chat) That is dummy data that is used as a placeholder for testing purposes otherwise the page won't display. I was planning on hooking up the state/flow afterwards once we have all the initial pages done for adult-child.